### ISSUE TITLE
Does the image classifier work with multichannel images? 

### ISSUE THREAD

**Issue by shaojus** [transformify-plugins/segmentify#39]
Does the segmentation work for multichannel images? It seems like for multichannel/3D data the classifier treats each layer as separate and segments each layer differently.

**Comment by sofroniewn** [comment]
hi @shaojus, I don't believe multichannel images are supported right now. I believe the only pretrained featurizer models are on 2D images right now and so we'd have to apply those to each plane separately, but one could combine all the features together in a single random forrest step. One could imaging fully 3D featurizers, but that would be more work and training. Let me know if you're interested in working on that. Segmentify isn't under active development right now - it is more of a proof of concept, but I'm happy to help.

**Comment by shaojus** [comment]
@sofroniewn yeah I've actually been working on trying to get segmentify to work for multichannel data recently and am interested in continuing to work on it.

**Comment by sofroniewn** [comment]
Great, if you want to make a PR let me know and i'll be happy to review
